lib/WP_Auth0_LoginManager.php

```diff
@@ -265,11 +265,11 @@ public function redirect_login() {
 
       $userinfo = json_decode( $response['body'] );
       if ( $this->login_user( $userinfo, $data->id_token, $data->access_token ) ) {
-        if ( null !== $stateFromGet && isset( $stateFromGet->interim ) && $stateFromGet->interim ) {
+        if ( ! empty( $stateFromGet->interim ) ) {
```

Comment:
I guess this is behaving like a dictionary, right?

```js
stateFromGet = { 
   interim: "some value",
   //....
}
```

In that case, could $stateFromGet be null / undefined and the call to obtain the value fail/throw? `$stateFromGet->interim`

Reply:
@lbalmaceda - `empty()` works in the case where the base object/array is undefined so:

<img width="408" alt="screenshot 2018-02-15 08 40 03" src="https://user-images.githubusercontent.com/855223/36268648-efd082f4-122b-11e8-933e-71845802e5fa.png">

Reply:
Ok, that's for `empty()`. But my question is: 
What if `$stateFromGet` is null/undefined. If you do `$stateFromGet->interim` will that throw an unhandled exception? (this operation is done before actually calling `empty()` as you need to resolve the parameter for the function)

Reply:
Nope, `empty()` handles it:

<img width="370" alt="screenshot 2018-02-15 09 15 20" src="https://user-images.githubusercontent.com/855223/36270595-f2977c40-1230-11e8-9c1a-c0a8bab545fb.png">